guides/common/assembly_managing-organizations.adoc

```diff
@@ -1,41 +1,4 @@
-[id="Managing_Organizations_{context}"]
-= Managing Organizations
-
-Organizations divide {ProjectName} resources into logical groups based on ownership, purpose, content, security level, or other divisions.
-ifdef::satellite[]
-You can create and manage multiple organizations through {ProjectName}, then divide and assign your Red Hat subscriptions to each individual organization.
-endif::[]
-ifndef::satellite[]
-You can create and manage multiple organizations through {ProjectName}, then divide and assign subscriptions to each individual organization.
-endif::[]
-This provides a method of managing the content of several individual organizations under one management system.
```

Comment:
Perhaps this should stay.
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`